eng/scripts/Language-Settings.ps1

```diff
@@ -67,7 +67,8 @@ function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
   }
   catch {
     # This is soft error and failure is expected for python metapackages
-    LogError "Failed to get all package properties"
+    LogError "Failed to get all package properties: $($_.Exception.Message)"
+    return $null
   }
```

Comment:
I would just remove the catch block.  It looks like I copied it from python and it's not really appropriate for rust:
> # This is soft error and failure is expected for python metapackages

This isn't a soft error in rust. `cargo metadata` should work and all of the code in the try block should succeed.

In this case, it's only there to ensure the caller returns back to their original location after the Push-Location